BUILD_MESON

```diff
@@ -39,7 +39,7 @@ By Default no action is required & meson configures everything needed to build D
 Note that the "eg++" or "clang++" package must be installed on OpenBSD as the default "g++"
 compiler is too old. Clang is a part of the base system in recent releases.
 
-in Meson you need to prepare "Buidling Dir". simply just type:
+in Meson you need to prepare "build directory". simply just type:
```

Comment:
Sentences should begin with a capital, "in Meson" -> "In Meson"

There needs to be "a" or "the" in front of "build directory".

Reply:
I send my commit or wait for your last reviews?

Reply:
I've finished looking at it now